index.html

```diff
@@ -53,7 +36,7 @@ <h2>Create new hero</h2>
                 </div>
             </div>
         </div>
-        <div class="subcol">
+        <div class="subcol colhero-list">

```

Comment:
how about `col-hero-list`  instead of `colhero-list`

Reply:
เปลี่ยนเรียบร้อยครับ